mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule CodeCorps.Mixfile do
   def project do
     [app: :code_corps,
      version: "0.0.1",
-     elixir: "1.3.4",
+     elixir: "1.4.0",
```

Comment:
Looks like the circleci is failing because of this, would undo for the PR as the upgrade to 1.4 should probably be in its own PR